src/Service/DynamoDb/src/Input/CreateTableInput.php

```diff
@@ -0,0 +1,426 @@
+    {
+        $this->GlobalSecondaryIndexes = $value;
+
+        return $this;
+    }
+
+    /**
+     * @param KeySchemaElement[] $value
+     */
+    public function setKeySchema(array $value): self
+    {
+        $this->KeySchema = $value;
+
+        return $this;
+    }
+
+    /**
+     * @param LocalSecondaryIndex[] $value
+     */
+    public function setLocalSecondaryIndexes(array $value): self
+    {
+        $this->LocalSecondaryIndexes = $value;
+
+        return $this;
+    }
+
+    public function setProvisionedThroughput(?ProvisionedThroughput $value): self
+    {
+        $this->ProvisionedThroughput = $value;
+
+        return $this;
+    }
+
+    public function setSSESpecification(?SSESpecification $value): self
+    {
+        $this->SSESpecification = $value;
+
+        return $this;
+    }
+
+    public function setStreamSpecification(?StreamSpecification $value): self
+    {
+        $this->StreamSpecification = $value;
+
+        return $this;
+    }
+
+    public function setTableName(?string $value): self
+    {
+        $this->TableName = $value;
+
+        return $this;
+    }
+
+    /**
+     * @param Tag[] $value
+     */
+    public function setTags(array $value): self
+    {
+        $this->Tags = $value;
+
+        return $this;
+    }
+
+    public function validate(): void
+    {
+        foreach ($this->AttributeDefinitions as $item) {
+            $item->validate();
+        }
+
+        if (null === $this->TableName) {
+            throw new InvalidArgument(sprintf('Missing parameter "TableName" when validating the "%s". The value cannot be null.', __CLASS__));
+        }
+
+        foreach ($this->KeySchema as $item) {
+            $item->validate();
+        }
+
+        foreach ($this->LocalSecondaryIndexes as $item) {
+            $item->validate();
+        }
+
+        foreach ($this->GlobalSecondaryIndexes as $item) {
+            $item->validate();
+        }
+
+        if (null !== $this->BillingMode) {
+            if (!BillingMode::exists($this->BillingMode)) {
+                throw new InvalidArgument(sprintf('Invalid parameter "BillingMode" when validating the "%s". The value "%s" is not a valid "BillingMode".', __CLASS__, $this->BillingMode));
+            }
+        }
+
+        if (null !== $this->ProvisionedThroughput) {
+            $this->ProvisionedThroughput->validate();
+        }
+
+        if (null !== $this->StreamSpecification) {
+            $this->StreamSpecification->validate();
+        }
+
+        if (null !== $this->SSESpecification) {
+            $this->SSESpecification->validate();
+        }
+
+        foreach ($this->Tags as $item) {
+            $item->validate();
+        }
+    }
+
+    /**
+     * @internal
+     */
+    private function requestBody(): array
+    {
+        $payload = [];
+
+        $index = -1;
+        foreach ($this->AttributeDefinitions as $mapValue) {
+            ++$index;
+            if (null !== $v = $mapValue) {
```

Comment:
thta's weird, it should not be null. Is it?

Reply:
This is not my change. But I reverted it now.

Reply:
fixed by #322